src/sql/workbench/contrib/welcome/page/browser/gettingStartedTour.css

```diff
@@ -0,0 +1,318 @@
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.ads_tour:focus {
+	outline: 0 none !important;
+}
+
+.monaco-workbench>.ads_tour {
+	position: absolute;
+	top: 0;
+	left: 0;
+	height: 100%;
+	width: 100%;
+	z-index: 9999;
+	font-size: 1em;
+	transition: font-size .25s;
+	outline: 0;
+}
+
+#workbench\.parts\.editor {
+	transition: filter .25s, opacity .2s;
+}
+
+.monaco-workbench.blur-background #workbench\.parts\.editor, .monaco-workbench.blur-background #workbench\.parts\.panel, .monaco-workbench.blur-background #workbench\.parts\.sidebar {
+	filter: blur(2px);
+	opacity: .4;
+}
+
+.hc-black .monaco-workbench.blur-background #workbench\.parts\.editor, .hc-black .monaco-workbench.blur-background #workbench\.parts\.panel, .hc-black .monaco-workbench.blur-background #workbench\.parts\.sidebar {
+	opacity: .2;
+}
+
+.monaco-workbench>.ads_tour>.key {
+	font-size: 1.6em;
+}
+
+.monaco-workbench>.ads_tour>.key>.label {
+	padding: 0 1ex;
+}
+
+.monaco-workbench>.ads_tour>.key>.shortcut {
+	letter-spacing: .15em;
+	font-size: .8125em;
+	font-family: "Lucida Grande", sans-serif;
+}
+
+.monaco-workbench>.ads_tour>.key.explorer {
+	position: absolute;
+	top: 13px;
+	left: 60px;
+}
+
+.monaco-workbench>.ads_tour>.key.search {
+	position: absolute;
+	top: 63px;
+	left: 60px;
+}
+
+.monaco-workbench>.ads_tour>.key.git {
+	position: absolute;
+	top: 113px;
+	left: 60px
+}
+
+.monaco-workbench>.ads_tour>.key.debug {
+	position: absolute;
+	top: 163px;
+	left: 60px;
+}
+
+.monaco-workbench>.ads_tour>.key.extensions {
+	position: absolute;
+	top: 213px;
+	left: 60px;
+}
+
+.monaco-workbench>.ads_tour>.key.watermark {
+	position: absolute;
+	top: calc(50% + 30px);
+	left: calc(50% + 30px);
+}
+
+.monaco-workbench>.ads_tour>.key.watermark>.label {
+	vertical-align: super
+}
+
+@media (max-height:500px) {
+	.monaco-workbench>.ads_tour>.key.watermark {
+		display: none;
+	}
+}
+
+.monaco-workbench>.ads_tour>.key.problems {
+	position: absolute;
+	bottom: 25px;
+	left: 45px;
+}
+
+.monaco-workbench>.ads_tour>.key.terminal {
+	position: absolute;
+	bottom: 25px;
+	left: 50%;
+}
+
+.monaco-workbench>.ads_tour>.key.notifications {
+	position: absolute;
+	bottom: 25px;
+	right: 16px;
+}
+
+.monaco-workbench>.ads_tour>.key.notifications>.label, .monaco-workbench>.ads_tour>.key.notifications>.shortcut, .monaco-workbench>.ads_tour>.key.problems>.label, .monaco-workbench>.ads_tour>.key.problems>.shortcut {
+	vertical-align: super;
+}
+
+.monaco-workbench>.ads_tour>.key.openfile {
+	position: absolute;
+	bottom: 25px;
+	right: 45px;
+}
+
+.monaco-workbench>.ads_tour>.key.openfile>.label {
+	vertical-align: super;
+}
+
+.monaco-workbench>.ads_tour>.key.commandPalette {
+	position: absolute;
+	top: 210px;
+	left: 0;
+	right: 0;
+	transition: top .25s, left .25s;
+	margin: auto;
+}
+
+.hide {
+	display: none;
```

Comment:
Where is this class used? I couldn't find it with a quick search. But is it possible to scope this down more? Generally it at least follows what I'd expect a class like hide to do but I still think we should strive to keep things logical in where they're defined. So if we're going to add a super generic rule like this then we should consider putting it in some general "common rules" css file instead